components/Chat/ChatInput.tsx

```diff
@@ -21,6 +26,13 @@ const ChatInput: React.FC<ChatInputProps> = ({
   return (
     <div className="w-full max-w-[555px] bg-white py-3 rounded-3xl border border-gray-300 p-1.5 mb-3 shadow-lg flex items-center">
       <form onSubmit={handleSubmit} className="w-full flex items-center">
+        {
+          profile.length > 0 ? (
+            <img src={`https://zora.co/api/avatar/${profile[0].address}`} alt="PFP" width={36} height={36} className="rounded-full" />
```

Comment:
I do not think 100% of profiles will be using a zora subdomain.

Instead of assuming all URL will be in this format, could you create a lib: `getZoraPfpLink`?
- `lib/zora/getZoraPfpLink` - check if URL starts with `/` if yes, return Zora URL prefix, otherwise, return the raw input url

Reply:
Yeah, right. All profile is not using zora subdomain. In my case, this uses other subdomain.
But it also has zora subdomain link. I mean it has both link.